src/ipx.ts

```diff
@@ -259,14 +262,6 @@ class IPX {
     }
   }
 
-  private extractImageFormat (format) {
-    format = format.split('.').shift()
-    if (format === 'jpg') {
```

Comment:
jpeg and jpg are same format. We would still need this normalizer somewhere otherwise there are two cache entries with same value